src/Pages/Wallets.razor

```diff
@@ -369,6 +377,102 @@
+                    </Column>
+                   <Column ColumnSize="ColumnSize.Is6">
+                           <Field>
+                               <FieldLabel>@($"{_sourceWalletName} Balance (this wallet)")</FieldLabel>
+                               <TextEdit @bind-Text="_sourceWalletBalance" PlainText Disabled />
+                           </Field>
+                    </Column>
+                    <Column ColumnSize="ColumnSize.Is6">
+                           <Field>
+                           <FieldLabel>@($"{_targetWalletName} Balance")</FieldLabel>
+                               <TextEdit @bind-Text="_transferTargetWalletBalance" PlainText Disabled />
+                           </Field>
+                    </Column>
+                    <Column ColumnSize="ColumnSize.Is6">
+                        <Validation Validator="args => ValidationHelper.ValidateWithdrawalAmount(args, _transferAllFunds)">
+                            <Field>
+                                <FieldLabel>Amount</FieldLabel>
+                                <FieldBody>
+                                    <NumericPicker TValue="decimal" Disabled="_transferAllFunds" Value="@_amountToTransfer" ValueChanged="(value) => { _amountToTransfer = value; }" CurrencySymbol="₿ " Max="@_maxWithdrawal" Min="@_minWithdrawal" Decimals="8">
+                                        <Feedback>
+                                            <ValidationError />
+                                        </Feedback>
+                                    </NumericPicker>
+                                </FieldBody>
+                                <FieldHelp>
+                                    @($"Current amount: {Math.Round(PriceConversionHelper.BtcToUsdConversion((decimal)_amountToTransfer, _btcPrice), 2)} USD")
+                                </FieldHelp>
+                            </Field>
+                        </Validation>
+                    </Column>
+                    <Column ColumnSize="ColumnSize.Is6">
+                            <Field>
+                                <FieldBody>
+                                    <Check TValue="bool" @bind-Checked="@_transferAllFunds">Transfer all funds</Check>
+                                </FieldBody>
+                            </Field>
+                    </Column>
+
+
+                </Validations>
+            </Fields>
+
+        </ModalBody>
+        <ModalFooter>
+            <Button Color="Color.Secondary" Clicked="@CloseAndCleanTransferFundsModal">Close</Button>
+            <Button Color="Color.Primary" Clicked="@TransferFunds">Transfer</Button>
+
+        </ModalFooter>
+    </ModalContent>
+</Modal>
+
+<ConfirmationModal
+    @ref="_multisigTransferModal"
+    Title="Transferring from a multisig wallet"
+    Body="You're trying to transfer from a multisig wallet. Please, go to Withdrawal to sign for this transfer."
+    CancelLabel="Yes, understood"
+    OnCancel="CloseMultisigTransferModal"
+/>
```

Comment:
I think the message should be more clear, like: "Your transfer request was successfully done, please, go to Withdrawals page to sign for this transfer."

also, as a next step, we could change this button for "Go to Withdrawals page" and send the user directly. But you'll need an additional cancel button.

Reply:
Fixed! Let's see if in the future Aleks ask for this :) or we cover in a frontend improvement session